base/deprecated.jl

```diff
@@ -612,6 +612,21 @@ end
 const MS_ASYNC = 1
 const MS_INVALIDATE = 2
 const MS_SYNC = 4
+
+for constant in (:MS_ASYNC, :MS_INVALIDATE, :MS_SYNC)
+    constant_str = string(constant)
+    @eval begin
+        @doc """
+            $($constant_str)
+
+        Enum constant for [`msync`](:func:`msync`). See your platform man page for details.
+        (not available on Windows).
+        """ ->
+        $constant
+    end
+end
```

Comment:
Rather than having 3 similar docstrings perhaps something like the following?

```julia
const MS_ASYNC = 1
const MS_INVALIDATE = 2
const MS_SYNC = 4

@doc """
    MS_ASYNC
    MS_INVALIDATE
    MS_SYNC

Enum constants for [`msync`](:func:`msync`). See your platform man page for details.
(not available on Windows).
""" ->
(MS_ASYNC, MS_INVALIDATE, MS_SYNC)
```

and then adjust the signature in the rst docs accordingly.

(`@doc` needed since this is inside a `@unix_only` block, not toplevel.)